app/src/main/java/com/example/dispatcher/MainActivity.kt

```diff
@@ -2,10 +2,22 @@ package com.example.dispatcher
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.ViewGroup
+import android.widget.Button
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        val crashButton = Button(this)
```

Comment:
Please remove this.
After you test that the crash is being reported to crashlytics we don't want this in our git.